### ISSUE TITLE
Icon picker editor renders the button "Add Item" not usable when editing a LandingPage in TheAgencyTheme

### ISSUE THREAD

**Issue by larremp** [OrchardCMS/OrchardCore#2661]
When changing the editor of the Icon Class field in the Service Content Type to Icon Picker, the "Add Item" button is not usable when trying to add a Service to the LandingPage content.

Checking the browser console shows a jquery "Uncaught TypeError:.."

![issueiconpicker](https://user-images.githubusercontent.com/27111713/48315687-13d3ab80-e59f-11e8-8058-96d36ed8f241.png)

**Comment by sebastienros** [comment]
Can you confirm the repro steps?

- Use the Agency recipe
- Change the editor for class to Icon Picker
- Edit the home page

We can't Add a service

**Comment by larremp** [comment]
Yes, the repro steps you outlined is correct. More specifically, I used the "Service" content type for this example.
